lib/node_modules/@stdlib/stats/base/dists/planck/pmf/test/fixtures/python/REQUIRE

```diff
@@ -0,0 +1,3 @@
+numpy 2.2.0
```

Comment:
What is this file? This should be removed. `REQUIRE` is a Julia thing, not Python. When in doubt, consult our snippets: https://github.com/stdlib-js/stdlib/tree/develop/tools/snippets/test/fixtures